test/smoke-limbo/no-loop-threads/Makefile

```diff
@@ -15,4 +17,4 @@ CC           = $(OMP_BIN) $(VERBOSE)
 
 include ../Makefile.rules
 run: $(TESTNAME)
-	$(RUNENV) ./$(TESTNAME) 2>&1 | $(FILECHECK) $(TESTSRC_MAIN)
+	$(RUNENV) $(RUNCMD)
```

Comment:
I think the machinery in the smoke will automatically construct this rule body when the `RUNCMD` is given. Basically, we can remove such `run` rules alltogether.

If this is not the case, I would argue that we should fix the machinery to make it behave this way.

Reply:
I agree that run rule can be removed completely and you should remove it so that you can also use the SUPPORTED / UNSUPPORTED functionality.